examples/event_emitter.rs

```diff
@@ -0,0 +1,28 @@
+use std::fs::File;
+
+use web_audio_api::context::{AudioContext, BaseAudioContext};
+use web_audio_api::node::{AudioNode, AudioScheduledSourceNode};
+
+fn main() {
+    env_logger::init();
+
+    let audio_context = AudioContext::default();
+
+    let file = File::open("samples/sample.wav").unwrap();
+    let buffer = audio_context.decode_audio_data_sync(file).unwrap();
+
+    let src = audio_context.create_buffer_source();
+    src.connect(&audio_context.destination());
+    src.set_buffer(buffer);
+
+    // @todo - should receive an event
```

Comment:
No sure what you mean, my comment was about that it still unfinished because the callback signature should be 
```rs
|e: Event| {
   println!("{:?}", event.type);
}
```
cf. https://dom.spec.whatwg.org/#interface-event

Reply:
Ah I understand now. Indeed the todo remains